docs/08-MultiPageDashApp.md

```diff
@@ -251,7 +251,45 @@ However, we recommend splitting out the page layouts into their own files in `pa
 
 ***
 
+**Query Strings**
 
+It's possible to pass query strings from the url to a layout function.
+For example:
+```python
+import dash
+
+dash.register_page(__name__, path='/dashboard')
+
+def layout(velocity=0, **other_unknown_query_strings):
```

Comment:
i wish we had a terser but still sufficiently clear way to handle this ** option.

Reply:
I agree, but for the first example in the docs, I think it's helpful.  Perhaps we can add a second example with something less verbose.